library/r-base

```diff
@@ -2,7 +2,7 @@ Maintainers: Carl Boettiger <rocker-maintainers@eddelbuettel.com> (@cboettig),
              Dirk Eddelbuettel <rocker-maintainers@eddelbuettel.com> (@eddelbuettel)
 GitRepo: https://github.com/rocker-org/rocker.git
 
-Tags: 3.6.3, latest
+Tags: 4.0.0, latest
 Architectures: amd64, arm64v8
-GitCommit: e76a46e4d082c0294437c34df1072f48c6fa3685
+GitCommit: 1286d1fa0165a06b93ea9c4cebd94a54cbf86227
 Directory: r-base
```

Comment:
Looks like the `Directory` value needs to be adjusted
```suggestion
Directory: r-base/4.0.0
```

Reply:
Good catch.

Background: I realized that I wanted to tag better (ie explicitly) so now there are 3.6.3 (from last time), latest (which I changed today) and 4.0.0 (which I changed last weekend to for now use experimental).  It didn't pick up here.  

I presume I can just point to latest/ hand have it tagged 4.0.0 and latest.  Or rather the way, point at 4.0.0/.  The Dockerfiles are identical now, but only one will remain 'frozen'.

Reply:
And silly me was scrathing my head as to why Travis CI failed, and I didn't think of this (as I made the change a few weeks ago).

Reply:
Me too; was trying to figure out why the travis script was looking at `refs/tags` until I asked Tianon (it couldn't find the commit so it tried our commit warehouse).